app/core/src/main/java/com/fsck/k9/preferences/SettingsImporter.java

```diff
@@ -269,6 +274,15 @@ public static ImportResults importSettings(Context context, InputStream inputStr
             }
 
             preferences.loadAccounts();
+
+            // create missing OUTBOX folders
+            for (Account account: preferences.getAccounts()) {
+                if (accountUuids.contains(account.getUuid())) {
+                    LocalStore localStore = DI.get(LocalStoreProvider.class).getInstance(account);
```

Comment:
Please move the dependency lookup outside of the loop.

Reply:
You need an `account` instance to be able to call `getInstance`

Reply:
Oh I see, done